fitnessllm_dataplatform/task_handler.py

```
@@ -0,0 +1,65 @@
+from os import environ
+
+import fire
+from cloudpathlib import GSClient
+
+from fitnessllm_dataplatform.entities.enums import DynamicEnum, FitnessLLMDataSource
+from fitnessllm_dataplatform.infrastructure.RedisConnect import RedisConnect
+from fitnessllm_dataplatform.stream.strava.entities.enums import StravaStreams
```

Ruff (0.8.2)

`fitnessllm_dataplatform.stream.strava.entities.enums.StravaStreams` imported but unused

Remove unused import: `fitnessllm_dataplatform.stream.strava.entities.enums.StravaStreams`

(F401)